internal/app/handlers.go

```diff
@@ -19,7 +19,10 @@ import (
 )
 
 func (app *Application) healthcheck(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("status: OK"))
+	_, err := w.Write([]byte("status: OK"))
+	if err != nil {
+		customErrors.ServerError(w, r, err, app.logger)
+	}
```

Comment:
@katyasoup also for when you're back: is this overly verbose or idiomatic? the linter complains about the unhandled errors, but in practice the app will catch panics and turn them into internal server errors. Our impression in the team meeting was this is just what go is like, but I'm curious for your more experienced take